quickget

```diff
@@ -1485,9 +1485,9 @@ function get_haiku() {
 
 function get_holoiso() {
     local HASH=""
-    local ISO="HoloISO_${RELEASE}_ux_changes-20230626_66f7c74061_bootchoice_fixes-1-x86_64.iso"
-    local URL="http://holoiso.itsvixano.me"
-
+    local ISO=$(wget -q -O- "https://api.github.com/repos/HoloISO/holoiso/releases" | sed 's/ /\n/g' | grep "HoloISO_${RELEASES}" | cut -d'\' -f1 | cut -d'/' -f4)
+    local URL="https://cd2.holoiso.ru.eu.org"
+    # Can't find hash
     HASH=$(wget -q -O- "${URL}/${ISO}.sha256sum" | cut -d' ' -f1)
```

Comment:
So please remove the fetching of the non-existent hash.
Also if they do not provide a checksum it is possible support my be removed if that is made a requirement.